src/node-file-trace.ts

```diff
@@ -116,6 +119,9 @@ export class Job {
     this.paths = resolvedPaths;
     this.log = log;
     this.mixedModules = mixedModules;
+    this.fileIOQueue = new Sema(fileIOConcurrency, {
+      capacity: fileIOConcurrency * 10,
```

Comment:
Why `fileIOConcurrency * 10`?

Reply:
Good point, we probably want to remove the `capacity` here as we can't know ahead of time a good amount that will be needed.